testing/armagetronad/APKBUILD

```diff
@@ -0,0 +1,45 @@
+# Maintainer: Díaz Urbaneja Diego <sodomon2@gmail.com>
+pkgname=armagetronad
+pkgver=0.2.8.3.4
+pkgrel=0
+pkgdesc="A Tron Clone in 3D."
+url="http://armagetronad.net/"
+arch="x86 x86_64"
+license="GPL"
+depends="ftgl libxml2 sdl_image sdl_mixer"
+makedepends="sdl_image-dev sdl_mixer-dev ftgl-dev libxml2-dev"
+subpackages="$pkgname-doc"
+options="!check"
+source="https://launchpad.net/$pkgname/0.2.8/$pkgver/+download/$pkgname-$pkgver.src.tar.gz"
+builddir="$srcdir/$pkgname-$pkgver"
+
+build() {
+cd "$builddir"
+  ./configure \
+    --prefix=/usr \
+    --disable-useradd \
+    --enable-dirty \
+    --enable-music \
+    --enable-armathentication \
+    --enable-master \
+    --enable-krawall \
+    --bindir=/usr/bin \
+    --sbindir=/usr/sbin \
+    --libexecdir=/usr/libexec \
+    --datadir=/usr/share \
+    --sysconfdir=/usr/share \
+    --sharedstatedir=/usr/share \
+    --localstatedir=/var \
+    --libdir=/usr/share \
+    --includedir=/usr/share \
+    --docdir=/usr/share/doc/armagetronad
+  make
+}
```

Comment:
The indention isn't correct here. Please use tools like apkbuild-lint from the atools package to avoid these mistakes.

Reply:
Apparently apkbuild-lint does not work, 
I've made several attempts but nothing works so I don't know how to fix the bleeding.